framework/db/ActiveQueryInterface.php

```diff
@@ -19,6 +19,8 @@
  * @author Qiang Xue <qiang.xue@gmail.com>
  * @author Carsten Brandt <mail@cebe.cc>
  * @since 2.0
+ *
+ * @property $link
```

Comment:
Property for interface ? wtf ?

Reply:
Yep, just for autocompletion.

Reply:
Interfaces does not have properties, this change is incorrect.

Reply:
Yeah, I know, I just did it for autocompletion and IDE was raising warning. I will remove it.